src/ui/Templates/UnderscoreTemplate.ts

```diff
@@ -34,7 +35,13 @@ export class UnderscoreTemplate extends Template {
     this.dataToString = (object) => {
       var extended = _.extend({}, object, UnderscoreTemplate.templateHelpers);
       if (this.template) {
-        return this.template(extended);
+        try {
+          return this.template(extended);
+        } catch (e) {
+          new Logger(this).error(l('CannotInstanciateTemplate', (element.id != '' ? element.id : 'ID undefined')), e.message, element);
```

Comment:
Don't need to output the ID of the element if you're already outputting the element itself.